simd/unit_tests/include/SIMDTesting_Utilities.hpp

```diff
@@ -135,8 +135,8 @@ class load_masked {
     for (std::size_t i = 0; i < n; ++i) {
       mask[i] = true;
     }
+    result = T(0);
     where(mask, result).copy_from(mem, Kokkos::Experimental::simd_flag_default);
-    where(!mask, result) = 0;
     return true;
```

Comment:
I'm not overly happy with this, but I couldn't stop the compiler from complaining differently.

Reply:
@ldh4 Any better idea?

Reply:
I think the change is fine, but I find it interesting that this loader is the only one getting flagged. Other loaders are also not explicitly initializing the value of `result` either.